static/js/server_events_dispatch.js

```diff
@@ -235,6 +234,7 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
         if (event.op === 'add') {
             _.each(event.subscriptions, function (rec) {
                 var sub = stream_data.get_sub_by_id(rec.stream_id);
+                sub.email_address = rec.email_address;
```

Comment:
We don't update `sub` from this file, `stream_data.js` is right place to add this code.

Reply:
Agreed!